osu.Game.Tests/Editing/Checks/CheckAudioQualityTest.cs

```diff
@@ -43,7 +42,7 @@ public void TestMissing()
 
             var mock = new Mock<IWorkingBeatmap>();
             mock.SetupGet(w => w.Beatmap).Returns(beatmap);
-            mock.SetupGet(w => w.Track).Returns((Track)null);
+            mock.SetupGet(w => w.Track).Returns(default(Track?).AsNonNull());
```

Comment:
Not really sure it's the right usage of `AsNonNull()` but i think it should be better than `(Track)null!` 🤔

Reply:
This definitely looks wrong.